src/prompts.ts

```diff
@@ -1,10 +1,7 @@
 import type { MessageParam } from "@anthropic-ai/sdk/resources";
 import { Outline, OutlineSection } from "./types";
 
-export function getTitleInferenceMessages(
-  primarySource: string,
-  description: string
-): MessageParam[] {
+export function getTitleInferenceMessages(primarySource: string, description: string): MessageParam[] {
```

Comment:
Is this Biome/prettier doing this? Maybe it's better to remove biome for now to get rid of the confusion